core/src/apps/hedera/sign_tx.py

```diff
@@ -0,0 +1,134 @@
+from trezor.crypto.curve import ed25519
+from trezor.messages import HederaSignTx
+from trezor.messages import HederaSignedTx
+from trezor.strings import format_amount
+
+from apps.common import paths
+from apps.common.keychain import auto_keychain
+
+from .layout import (
+    confirm_account,
+    confirm_associate_token,
+    confirm_create_account,
+    confirm_transfer_hbar,
+    confirm_transfer_token,
+    format_account,
+)
+
+transaction_types = {
+    "CONFIRM": "Confirm Account",
+    "CREATE": "Create Account",
+    "TOKEN": "Token Transfer",
+    "HBAR": "Hbar Transfer",
+    "ASSOCIATE": "Token Associate",
+}
+
+
+def validate_tx(msg: HederaSignTx) -> str | None:
+    if msg.tx is not None:
+        # Create Account
+        if msg.tx.cryptoCreateAccount is not None:
```

Comment:
This also needs to check that `msg.tx.cryptoCreateAccount.initialBalance is not None`